nondex-maven-plugin/src/it/inexistent-variable-argline-it/verify.groovy

```diff
@@ -0,0 +1,3 @@
+File nondexDirectory = new File( basedir, ".nondex" );
+
+assert ! nondexDirectory.exists();
```

Comment:
No. Because by default the IT expects that the result of the build is PASS. If whatever the invoker.properties specifies should be ran fails (or more generally does not have the expected outcome because the expected outcome could be that the command should result in failure) then verify.groovy is not even invoked.